schemas/documents/registry/landStewardsPage.js

```diff
@@ -0,0 +1,69 @@
+export default {
+  name: 'landStewardsPage',
+  type: 'document',
+  title: 'For Land Stewards Page',
+  __experimental_actions: ['update', 'create', 'delete', 'publish'],
+  fields: [
+    {
+      name: 'heroSection',
+      type: 'heroSection',
+      title: 'Hero Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'designedForFarmersSection',
+      type: 'imageItemsSection',
+      title: 'Designed for Farmers Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'joinFarmersSection',
+      type: 'dualImageSection',
+      title: 'Join Farmers Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'practicesOutcomesSection',
+      type: 'practicesOutcomesSection',
+      title: 'Practices and Outcomes Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'timelineSection',
+      type: 'timelineSection',
+      title: 'Timeline Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'featuredSection',
+      type: 'reference',
+      to: [{type: 'featuredSection'}],
+      title: 'Featured Credit Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'moreQuestionsSection',
+      type: 'bottomBanner',
+      title: 'More Questions Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'footerButtonText',
+      type: 'string',
+      title: 'Footer Button Text',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'footerLink',
+      type: 'url',
+      title: 'Footer Button Link',
+      description: 'This opens in a modal',
+      validation: Rule => Rule.required(),
+    },
```

Comment:
you can just use existing `button` object here instead

Reply:
done